eclipse/sw360
# feat(ReleaseUi): Add other / detected license in release

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.
 * PR to display other license from CLI file and add it to `other license` attribute of `release`.
 * Which issue is this pull request belonging to and how is it solving it? #1051 
 * Did you add or update any new dependencies that are required for your change? - `No`

Issue: #1051 

### Suggest Reviewer
> You can suggest reviewers here with an @mention.

### How To Test?
> How should these changes be tested by the reviewer?
 * Attach CLI files with Main/Other license ids to a release.
 * Go to `Clearing Details` tab in `Release Details` page, and click on `Show License Info` button in `SPDX Attachments` table.
 * Clicking on `Show License Info` button should load all the licenses from the CLI.
 * Click on `Add data to this release` button to add the Main/License ids to the release.
> Have you implemented any additional tests? - `No`

### Checklist
Must:

Signed-off-by: Abdul Kapti <abdul.kapti@siemens-healthineers.com>
